db/flush_job.cc

```diff
@@ -80,22 +80,24 @@ const char* GetFlushReasonString (FlushReason flush_reason) {
   }
 }
 
-FlushJob::FlushJob(
-    const std::string& dbname, ColumnFamilyData* cfd,
-    const ImmutableDBOptions& db_options,
-    const MutableCFOptions& mutable_cf_options, const uint64_t* max_memtable_id,
-    const FileOptions& file_options, VersionSet* versions,
-    InstrumentedMutex* db_mutex, std::atomic<bool>* shutting_down,
-    std::vector<SequenceNumber> existing_snapshots,
-    SequenceNumber earliest_write_conflict_snapshot,
-    SnapshotChecker* snapshot_checker, JobContext* job_context,
-    LogBuffer* log_buffer, FSDirectory* db_directory,
-    FSDirectory* output_file_directory, CompressionType output_compression,
-    Statistics* stats, EventLogger* event_logger, bool measure_io_stats,
-    const bool sync_output_directory, const bool write_manifest,
-    Env::Priority thread_pri, const std::shared_ptr<IOTracer>& io_tracer,
-    const std::string& db_id, const std::string& db_session_id,
-    std::string full_history_ts_low)
+FlushJob::FlushJob(const std::string& dbname, ColumnFamilyData* cfd,
+                   const ImmutableDBOptions& db_options,
+                   const MutableCFOptions& mutable_cf_options,
+                   uint64_t max_memtable_id, const FileOptions& file_options,
```

Comment:
The only change here is from `const uint64_t* max_memtable_id` to `uint64_t max_memtable_id`. The rest results from auto-formatting.